app/Http/Middleware/CheckAdmin.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+
+class CheckAdmin
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure(\Illuminate\Http\Request): (\Illuminate\Http\Response|\Illuminate\Http\RedirectResponse)  $next
+     * @return \Illuminate\Http\Response|\Illuminate\Http\RedirectResponse
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        $user = $request->user();
+        if ($user && $user->isAdmin) {
+            return $next($request);
+        }
+        return redirect('/');
```

Comment:
```suggestion
        }
        
        return redirect('/');
```
1 line trống trc return

Reply:
Đã fix